astroquery/esa/hsa/tests/dummy_handler.py

```diff
@@ -3,7 +3,7 @@
 __all__ = ['DummyHandler']
 
 
-class DummyHandler(object):
+class DummyHandler():
```

Comment:
The following would be equivalent, but shorter:
```suggestion
class DummyHandler:
```

Reply:
oh, sure. Thanks!